demo-166-storage-api-webapis - Done

ade1705 opened the pull request:
# Description

Implemented demo for Selection API

Fixes [166](https://github.com/atapas/webapis-playground/issues/166)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/atapas/webapis-playground/blob/master/HOW-TO-ADD-DEMO.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

atapas commented:
@ade1705 Hey please resolve the conflict so that I can start review?

ade1705 commented:
> @ade1705 Hey please resolve the conflict so that I can start review?

@atapas conflict resolved

atapas reviewed (changes requested):
@ade1705 Do not use alerts. Let's have a section in the page to show the success and error messages.

ade1705 commented:
> @ade1705 Do not use alerts. Let's have a section in the page to show the success and error messages.

@atapas I removed the alerts and added a section in the page to show the messages
